src/cascadia/TerminalControl/SearchBoxControl.cpp

```diff
@@ -4,6 +4,7 @@
 #include "pch.h"
 #include "SearchBoxControl.h"
 #include "SearchBoxControl.g.cpp"
+#include <LibraryResources.h>

```

Comment:
since you are not using `RS_` you do not need this file

Reply:
removed. I think I once used RS_ in this file but then moved it.